mmv1/products/dataprocgdc/ServiceInstance.yaml

```
@@ -0,0 +1,123 @@
+base_url: projects/{{project}}/locations/{{location}}/serviceInstances
+create_url: projects/{{project}}/locations/{{location}}/serviceInstances?serviceInstanceId={{service_instance_id}}
+self_link: projects/{{project}}/locations/{{location}}/serviceInstances/{{service_instance_id}}
+id_format: projects/{{project}}/locations/{{location}}/serviceInstances/{{service_instance_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/serviceInstances/{{service_instance_id}}
+name: ServiceInstance
+description: A service instance is an instance of the Dataproc operator running on a GDC cluster.
+autogen_async: true
+properties:
+  - name: gdceCluster
+    type: NestedObject
+    properties:
+      - name: gdceCluster
+        type: String
+        description: 'Required. Gdce cluster resource id. '
+        required: true
+    description: 'Gdce cluster information. '
+  - name: name
+    type: String
+    description: 'Identifier. The name of the service instance. '
+    output: true
+  - name: uid
+    type: String
+    description: "Output only. System generated unique identifier for this service instance,
+      formatted as\nUUID4. "
+    output: true
+  - name: displayName
+    type: String
+    description: 'Optional. User-provided human-readable name to be used in user interfaces. '
```

Optional / required will be handled automatically during docs generation based on whether the field is optional or required.

```suggestion
    description: 'User-provided human-readable name to be used in user interfaces. '
```

---

Done